client/src/pages/AssignMates/AssignMatesPage.tsx

```diff
@@ -80,8 +181,27 @@ const AssignMatesPage = (): JSX.Element => {
     setGroups(newGroup);
   };
 
-  const submitNewAssignments = () => {
-    // TODO send to backend here
+  /**
+   * Submit mate assignments to server
+   */
+  const submitNewAssignments = async () => {
+    const mateAssignments: MateAssignmentInput[] = [];
+    Object.keys(groups)
+      .filter((familyId) => familyId !== unassignedFamilyId)
```

Comment:
We're only pulling from unassigned family, so no need to reassign mate to unassigned again.